spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -586,16 +586,32 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
   }
 
   /**
-   * Some other modules point to current module
+   * Build graph: some other modules point to current module
    * @param nodes upstream module nodes
    * @return node containing current module
    */
   def inputs(nodes : ModuleNode[T]*): ModuleNode[T] = {
     require(this.isInstanceOf[AbstractModule[_, Tensor[T], T]],
-      "AbstractModule: Only module with tensor output can be added into a graph node")
-    val curNode = new ModuleNode[T](this.asInstanceOf[AbstractModule[Activity, Tensor[T], T]])
+      s"AbstractModule: ${this.getClass.getName} has multiple outputs, please use (module, index)" +
```

Comment:
use module[i]?

Reply:
Because of the type erase, this will be always true.. Have removed this, as we support connect a table to a node.